src/htrun/README.md

```diff
@@ -0,0 +1,1208 @@
+# Development moved
```

Comment:
We'll need to fix up this readme to remove outdated information.

Reply:
In further cleanup, we will update the `Readme` and raise a separate PR